barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -49,70 +49,6 @@ class ClientIVCTests : public ::testing::Test {
     }
 };
```

Comment:
I think a test suite should start with the most basic test so I moved the serialisation tests at the bottom of the file